### ISSUE TITLE
Releases-by-author page shows multiple copies of the same release

### ISSUE THREAD

**Issue by Nemo157** [rust-lang/docs.rs#1852]
On [my releases page](https://docs.rs/releases/Nemo157) some crates like `git-appraise` are repeated multiple times. Seems to be correlated with those releases having had multiple builds performed.

**Comment by syphar** [comment]
Yep, this is definitely related to multiple builds for a single release.

**Comment by syphar** [comment]
I did some test and most solutions in the SQL query I tried actually slow down the query too much. 

The real solution is probably to store the `latest_build_id` on the `releases` row, the same way we store `latest_release_id` on the `crates` row. 

For now I'll leave it and focus on other issues.

**Comment by syphar** [comment]
@Nemo157 I believe this can still happen on all other release-pages since the join to the `builds` table is in all pages. 

Or am I missing something?

**Comment by Nemo157** [comment]
Ah yes, it can happen on other pages, but probably never will because there's no chance of use requeuing and rebuilding a release within 25 builds (or whatever the limit is).
